src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -136,6 +152,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            // in_array() will only be optimized if the third argument is true
+            if ('in_array' === $lowerFunctionName && $this->configuration['opcache-only']) {
+                $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $next);
+                if ('true' !== $tokens[$tokens->getPrevMeaningfulToken($end)]->getContent()) {
```

Comment:
Not entirely sure if this makes sense, but what do you think about lower-casing the content, just in case someone does not use the corresponding fixer for native constants and decides to write code like this:

```php
if (in_array($needle, $haystack, TrUe) {

}
```

Reply:
Very good remark. Changed in 057b3dc30d33eb57371fdb0b04790173c5b80b2d.